product_docs/docs/pem/10/pem_rel_notes/src/pem_10.1.0_rel_notes.yml

```diff
@@ -0,0 +1,279 @@
+  type: Enhancement
+  jira: PEM-5555
+  impact: low
+
+- relnote: Fixed an issue whereby EFM probes would fail due to a JSON data parser exception.
+  type: Bug Fix
+  jira: PEM-5540
+  addresses: 46527
+  impact: medium
+
+- relnote: Fixed an issue where date in Alert Status table yields year as 1970.
+  type: Bug Fix
+  jira: PEM-5565
+  impact: medium
+
+- relnote: Fixed an issue with the streaming replication chart where the user would see
+    an error `Column "Cluster Name" does not exist`.
+  type: Bug Fix
+  jira: PEM-5542
+  addresses: 47332
+  impact: medium
+
+- relnote: Fixed an issue in Email Groups whereby the delete button would be enabled on the default group when it should be disabled.
+  type: Bug Fix
+  jira: PEM-5527
+  impact: low
+
+- relnote: Fixed an issue whereby an error in a dashboard would prevent all subsequently accessed dashboards from loading correctly.
+  type: Bug Fix
+  jira: PEM-5543
+  impact: high
+
+- relnote: Fixed bug that would cause 'duplicate key' errors in browser console logs.
+  type: Bug Fix
+  jira: PEM-5510
+  impact: low
+
+- relnote: Fixed an issue which could result in an error `The error: module 'lib' has no attribute 'X509_V_FLAG_NOTIFY_POLICY'` during PEM configuration.
+  type: Bug Fix
+  jira: PEM-5546
+  impact: medium
+
+- relnote: Fixed an issue with the webhook 'test' button state whilst waiting for a response.
+  type: Bug Fix
+  jira: PEM-5507
+  impact: low
+
+- relnote: Added pywinpty package to dependencies on Windows.
+  type: Change
+  jira: PEM-5466
+  impact: low
+
+- relnote: Fixed an issue whereby a "threshold" key error was occurring on update of Alert fields other than threshold values.
+  type: Bug Fix
+  jira: PEM-5548
+  impact: medium
+
+- relnote: Jobs which are created by an alert are no longer listed in the PEM object explorer.
+  type: Change
+  jira: PEM-5151
+  details: 'Jobs created by alerts were previously visible in object explorer. This led to confusion as users were able to edit these jobs in unintended ways which would typically result in breakage.'
+  impact: low
+
+- relnote: Fixed a PEM configuration failure on NGINX/uWSGI when UMASK is 0027
+  type: Bug Fix
+  jira: PEM-5575
+  impact: low
+
+- relnote: Fixed an issue whereby NGINX could not be started when ipv6 was disabled
+  type: Bug Fix
+  jira: PEM-5644
+  impact: low
+
+- relnote: Fixed an issue whereby psycopg errors would occur while authenticating PEM using kerberos on RHEL 9
+  type: Bug Fix
+  jira: PEM-5523
+  details: PEM is now fully compatible with Kerberos on RHEL 9, so users of this combination are recommended to upgrade to PEM 10.1.
+  impact: high
+
+- relnote: Fixed an issue whereby Alerts were not saved when execute_alert_on was toggled.
+  type: Bug Fix
+  jira: PEM-5593
+  impact: low
+
+- relnote: Fixed an issue with redirects after editing a dashboard.
+  type: Bug Fix
+  jira: PEM-5594
+  impact: low
+
```

Comment:
Two more fixes we did, which are missing from here-
| PEM-5596 - Breadcrumb tree not displaying properly when dashboard is opened from the context menu
| PEM-5600 - Fixed the missing teams field from the Create/Edit server dialogue. Also added the privilege check while registering the server.